roles/kube-node-common/tasks/kubelet.yml

```diff
@@ -67,18 +67,28 @@
     repo: "deb https://apt.kubernetes.io/ kubernetes-{{ kubernetes_repo_distribution }} main"
     state: present
 
-- name: Making sure kubelet, kubectl, kubeadm, cri-tools are installed
+- name: Making sure kubelet, kubectl, kubeadm are installed
   apt:
     name:
       - openssl
       - kubelet={{ kubelet_version }}-00
       - kubeadm={{ kubeadm_version }}-00
       - kubectl={{ kubectl_version }}-00
-      - cri-tools={{ critools_version }}-00
       - kubernetes-cni={{ kubernetescni_version }}-00
     state: present
     update_cache: yes
 
+- name: Install cri-tools
+  become: yes
```

Comment:
Shouldn't this be already set from the play?

Reply:
became flag?

Reply:
Yes, I was referring to the `become` flag.

Reply:
Ok, so every become flag should be removed inside the role

Reply:
yes please, let's move all the `become flags` to the playbooks

Reply:
Done!